lib/ktl/migration_plan.rb

```diff
@@ -2,10 +2,13 @@
 
 module Ktl
   class MigrationPlan
-    def initialize(zk_client, old_leader, new_leader)
+    def initialize(zk_client, old_leader, new_leader, options = {})
       @zk_client = zk_client
       @old_leader = old_leader.to_java
       @new_leader = new_leader.to_java
+      @options = options
```

Comment:
It doesn't look like you're using `@options` except for in the constructor so it seems unnecessary to keep it around.